otel-config.yaml

```diff
@@ -11,6 +11,12 @@ receivers:
 
 processors:
   batch:
+  memory_limiter:
+    # 75% of maximum memory up to 4G
+    limit_mib: 1536
```

Comment:
Did the comment mean to say 2gb or should this limit change?

Reply:
The limiter is there following one of the configuration examples which I can no longer find.

Full docs for the memory limiter processor: https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md

The exact values I think depend on the machine that this would be running on. We can also set the percentage values, which would scale with the available memory on whatever machine it runs on, assuming it's a linux one with cgroups support.